.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_JWT_SECRET="iloveyou"
```

Comment:
you should put .env file in gitignore

Reply:
I will do this, need to fix hosting also accordingly